dip-0022.md

```diff
@@ -77,7 +77,7 @@ the following structure:
 
 There are two possible ways to form a ChainLock: using a single LLMQ as per
 DIP0008 and using multiple LLMQs. We use `SPORK_19_CHAINLOCKS_ENABLED` spork
-to signal which method should be used by the network: `0` - single LLMQ,
+to signal which method should be used by the network:`0` - single LLMQ,
```

Comment:
why this one?

Reply:
Oops. I originally moved the two items into a list and apparently deleted the space there when I decided to undo that change. I think I transposed the two lines when I did the PR and thought I was adding a space rather than removing one :see_no_evil: 
```suggestion
to signal which method should be used by the network: `0` - single LLMQ,
```